hollow-ui-tools/src/main/java/com/netflix/hollow/ui/HollowUISession.java

```diff
@@ -85,24 +87,23 @@ public static HollowUISession getSession(HttpServletRequest req, HttpServletResp
     }
 
     static {
-        Thread sessionCleanupThread = new Thread(new Runnable() {
-            public void run() {
-                Iterator<Map.Entry<Long, HollowUISession>> iter = sessions.entrySet().iterator();
-                while(iter.hasNext()) {
-                    Map.Entry<Long, HollowUISession> entry = iter.next();
-                    if(entry.getValue().lastAccessed + SESSION_ABANDONMENT_MILLIS < System.currentTimeMillis())
-                        iter.remove();
-                }
-
-                try {
-                    Thread.sleep(60000L);
-                } catch (InterruptedException e) { }
-            }
-        });
-
+        Thread sessionCleanupThread = new Thread(HollowUISession::cleanupSessions, "hollow | ui-session-cleanup");
         sessionCleanupThread.setDaemon(true);
         sessionCleanupThread.start();
     }
 
+    private static void cleanupSessions() {
+        while (true) {
+            Iterator<Map.Entry<Long, HollowUISession>> iter = sessions.entrySet().iterator();
```

Comment:
This can be shrunk down to:
```java
sessions.values()
    .removeIf(s-> s.lastAccessed + SESSION_ABANDONMENT_MILLIS < System.currentTimeMillis());
```

Reply:
Did it.